cmd/manager/main.go

```diff
@@ -165,10 +160,6 @@ func main() {
 			log.Error(fmt.Errorf("timed out waiting for main cache to sync"), "")
 			os.Exit(1)
 		}
-		if !idlerClientCache.WaitForCacheSync(stopChannel) {
```

Comment:
We don't need this?

Reply:
no, because we don't need the all-namespaces client (cache) in any of the subsequent "commands":
https://github.com/codeready-toolchain/member-operator/blob/17781a03f5505e6161ad1ef44cf6754a9d278e64/cmd/manager/main.go#L163-L173

Reply:
ah right, I did not realize it was not necessary for the subsequent commands 🤦‍♂️